src/main/java/space/obminyashka/items_exchange/dto/UserWithoutChildrenDto.java

```diff
@@ -0,0 +1,53 @@
+package space.obminyashka.items_exchange.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+import space.obminyashka.items_exchange.model.enums.Status;
+import space.obminyashka.items_exchange.util.PatternHandler;
+
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
+@Setter
+public class UserWithoutChildrenDto {
+
+    @ApiModelProperty(required = true)
+    @NotEmpty(message = "{invalid.not-empty}")
+    @Size(min = 2, max = 50, message = "{invalid.size}")
+    private String username;
+    @ApiModelProperty(required = true)
+    @NotEmpty(message = "{invalid.not-empty}")
+    @Email(message = "{invalid.email}")
+    private String email;
```

Comment:
You added such fields `email`, `online`, `avatarImage`, `lastOnlineTime`, `status`, `updated` but never used any of them. So why did you add those fields?